packaging/DISTRIBUTIONS.md

```diff
@@ -7,31 +7,245 @@ The following table is a work in progress. We have concluded on the list of dist
 that we currently supporting and we are working on documenting our current state so that our users
 have complete visibility over the range of support.
 
-Distribution | Family | Architecture | Code health | Installer support | Kickstart support | Binary packaging support | Integrity testing (CI) | Functionality testing (CI) | Community support
-:------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :------------------: | :--------------------
-14.04.6 LTS (Trusty Tahr) | Ubuntu |  |  |  |  |  |  | | 
-16.04.6 LTS (Xenial Xerus) | Ubuntu |  |  |  |  |  |  | | 
-18.04.2 LTS (Bionic Beaver) | Ubuntu |  |  |  |  |  |  | | 
-19.04 (Disco Dingo) Latest | Ubuntu |  |  |  |  |  |  | | 
-Debian 7 (Wheezy) | Debian |  |  |  |  |  |  | | 
-Debian 8 (Jessie) | Debian |  |  |  |  |  |  | | 
-Debian 9 (Stretch) | Debian |  |  |  |  |  |  | | 
-Debian 10 (Buster) | Debian |  |  |  |  |  |  | | 
-Versions 6.* | RHEL |  |  |  |  |  |  | | 
-Versions 7.* | RHEL |  |  |  |  |  |  | | 
-Versions 8.* | RHEL |  |  |  |  |  |  | | 
-Fedora 28 | Fedora |  |  |  |  |  |  | | 
-Fedora 29 | Fedora |  |  |  |  |  |  | | 
-Fedora 30 | Fedora |  |  |  |  |  |  | | 
-Fedora 31 | Fedora |  |  |  |  |  |  | | 
-CentOS 6.* | Cent OS | |  |  |  |  |  |  | | 
-CentOS 7.* | Cent OS | |  |  |  |  |  |  | | 
-CentOS 8.* | Cent OS | |  |  |  |  |  |  | | 
-Open SuSE Leap 15.0 | Open SuSE |  |  |  |  |  |  | | 
-Open SuSE Leap 15.1 | Open SuSE |  |  |  |  |  |  | | 
-Open SuSE Tumbleweed (latest) | Open SuSE |  |  |  |  |  |  | | 
-SuSE Enterprise Linux Server 11 | SLES |  |  |  |  |  |  | | 
-SuSE Enterprise Linux Server 12 | SLES |  |  |  |  |  |  | | 
-SuSE Enterprise Linux Server 15 | SLES |  |  |  |  |  |  | | 
-Arch Linux (latest) | Arch |  |  |  |  |  |  | | 
-All other linux | Other | |  |  |  |  |  | | 
+**Legend**:
+
+- **Version**: Operating System Version supported
+- **Family**: The family that the OS belongs to
+- **CI Smoke Testing**: Smoke testing has been implemented on our CI, to prevent broken code reaching our users
+- **Ci Testing**: Testing has been implemented, to prevent broken or problematic code reaching our users
```

Comment:
Capitalize the `I` in `CI`. Also, could we just say "bugs" instead of "broken or problematic code"?

```suggestion
- **CI Testing**: Testing has been implemented to prevent broken or problematic code reaching our users
```